.github/workflows/test.yml

```diff
@@ -0,0 +1,110 @@
+name: Test
+on:
+  push:
+    branches:
+      - v2-preview
```

Comment:
I *think* this will work if you specify all branches, as this workflow will only exist in the v2-preview branch anyway until it's merged to master.

Reply:
Doesn't work 🤨

Tried it with a few combinations and 6 workflows always end up running. It will kick off runs for the PR and the commit made to the current branch `konradpabjan/v2-is-almost-here`